### PR TITLE
Remove install2.r from dockerfile and test dockerhub push action

### PR DESCRIPTION
**Pull request by jashapiro**
## Purpose:
This PR is primarily designed to test #238 (automatic pushing to dockerhub) , but along the way, it closes https://github.com/AlexsLemonade/refinebio-examples/issues/235

## Strategy
I removed calls to `install2.R` and replaced them with the more verbose, but built-in `install.packages()` function. The reasoning here is that this should result in a somewhat smaller docker image, as `install2.R` seems to retain the cached versions of packages, rather than deleting them.

## Concerns/Questions for reviewers:
Did I miss any important packages?

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -33,39 +33,50 @@ RUN pip3 install "setuptools==49.3.0" "six==1.15.0" "wheel==0.34.2"
...
 # org.Mm.eg.db is required for gene mapping
-RUN install2.r --error --deps TRUE \
-    ggfortify \
-    ggsignif \
-    patchwork \
-    pheatmap \
-    Rtsne \
-    umap  \
-    VennDiagram
+RUN Rscript -e  "install.packages( \
```
Comment (jaclyn-taroni) [inline docker/Dockerfile:67]: Can you remove the outdated comment above this line?

**Review by jaclyn-taroni — APPROVED** [review]
👍

File: docker/Dockerfile [hunk 2]
```diff
@@ -33,39 +33,50 @@ RUN pip3 install "setuptools==49.3.0" "six==1.15.0" "wheel==0.34.2"
...
+      'limma', \
+      'marray', \
+      'org.Mm.eg.db', \
+      'org.Dr.eg.db'), \
+     update = FALSE)"
 
 # Installs packages needed for plottings
 # treemap, interactive plots, and hex plots
 # Rtsne and umap are required for dimension reduction analyses
 # org.Mm.eg.db is required for gene mapping
```
Comment (jashapiro) [inline docker/Dockerfile:66]:
```suggestion
```

File: docker/Dockerfile (hunk 1 above)
Comment (jashapiro) [reply docker/Dockerfile:66]: I was doing that as you reviewed! With those exact words.

**Pull request merged**